### PR TITLE
Improve first run instructions

### PR DESCRIPTION
**Pull request by rogeriochaves**
I found some difficulties trying to run the project, so I tried to improve that for whomever wants to contribute to the project next

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -15,3 +15,5 @@ services:
       - "17474:7474"
     volumes:
       - ./db/dbms:/data/dbms
+    environment:
+      - NEO4J_AUTH=none
```
Comment (rogeriochaves) [inline docker-compose.yml]: its easier than configuring a password manually, and not a problem when running locally

File: wsgi-conf.ini [hunk 2]
```diff
@@ -5,3 +5,4 @@ master = true
 processes = 5
 
 socket = 0.0.0.0:5100
+http = 0.0.0.0:5000
```
Comment (rogeriochaves) [inline wsgi-conf.ini]: apparently there was nothing really running on port 5000, and when I tried to use port 5100 I was getting this error: https://stackoverflow.com/questions/15878176/uwsgi-invalid-request-block-size

Comment (Eik-S) [reply wsgi-conf.ini]: This is the production configuration to be used with nginx. therefor http is not necessary and port 5000 is still usable through the docker-compose.yml directly linking to the flask api

Comment (rogeriochaves) [reply wsgi-conf.ini]: I just noticed the problem, this actually doesn't work:
```
CMD python server.py runserver -h 0.0.0.0
CMD uwsgi --ini wsgi-conf.ini
```
on docker we can only ever have a single CMD, if we put two docker doesn't run them both, instead the last one overrides all others

so I'll change it to this:
`CMD uwsgi --ini wsgi-conf.ini & python server.py runserver -h 0.0.0.0`
this way we have uwsgi on the background and python flask on the foreground

File: docker-compose.prod.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+version: '3'
+services:
+  web:
+    container_name: tracemap-api
+    build: .
+    ports:
+     - '5000:5000'
+     - '5100:5100'
+    env_file: .env
```
Comment (rogeriochaves) [inline docker-compose.prod.yml]: I've created a separate file for prod config, so we leave it easy for developers to develop with a simple `docker-compose up`, while still being able to deploy with `docker-compose up -f docker-compose.prod.yml`

**Pull request merged**